framework/src/executioners/Eigenvalue.C

```diff
@@ -52,13 +52,12 @@ Eigenvalue::validParams()
       "normal_factor", 1.0, "Normalize eigenvector to make a defined norm equal to this factor");
 
   params.addParam<bool>("auto_initialization",
-                        false,
+                        true,
                         "If true, we will set an initial eigen vector in moose, otherwise EPS "
                         "solver will initial eigen vector");
 
-  params.addParam<bool>("newton_inverse_power",
-                        true,
-                        "If Newton and Inverse Power is combined in SLEPc side");
+  params.addParam<bool>(
```

Comment:
Why is this parameter?  I thought for the slepc solver inside Eigenvalue::execute, we always have the combined newton and inverse power. On the other hand, the solver in Eigenvalue::init, we only want to have fixed number of inverse power iterations.

Reply:
This parameter is added when we want SLEPc to do everything instead of separating `power` and `newton`.

I believe  Grifin will not touch this parameter

Reply:
addprivateparam